cmd/omni/cmd/cmd.go

```diff
@@ -607,13 +607,10 @@ var initOnce = sync.OnceValue(func() *cobra.Command {
 		"enable Stripe machine usage reporting",
 	)
 
-	rootCmd.Flags().BoolVar(
-		&config.Config.DisableLegacyJoinTokens,
-		"disable-legacy-join-tokens",
-		config.Config.DisableLegacyJoinTokens,
-		"disables joining the machines running Talos below 1.6 "+
-			"by forcing strict policy that uses join token only for the initial machine join, "+
-			"then uses randomly generated secret stored in the machine META partition",
+	rootCmd.Flags().Var(
+		&config.Config.JoinTokensMode,
+		"join-tokens-mode",
+		"configures Talos machine join flow to use secure node tokens",
```

Comment:
Maybe let's print here the possible values as well...

Reply:
it prints them actually, as it's a custom Flag implementation.
That's how it looks like in the CLI:
```
--join-tokens-mode [legacy,legacyAllowed,strict]          configures Talos machine join flow to use secure node tokens (default legacy)
```